core_impl_android/src/main/java/com/google/instrumentation/stats/StatsManagerImpl.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.stats;
+
+import com.google.instrumentation.common.SimpleEventQueue;
```

Comment:
SimpleEventQueue is used for testing, so it needs to be in core_impl.  I just used it in the Android implementation as a placeholder.  Do you think I should copy it and rename the original class to something like TestEventQueue?